<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAboutDialog</class>
 <widget class="QDialog" name="FAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>313</width>
    <height>190</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="headerImage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>57</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>57</width>
         <height>57</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/images/icon.png</pixmap>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="headerText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>57</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;FLIST_NAME&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;FLIST_VERSIONNUM&lt;/span&gt;&lt;br/&gt;Using Qt QT_VERSION_STR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;By &lt;/span&gt;&lt;a href=&quot;https://f-list.net/c/Viona&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Viona&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;, &lt;/span&gt;&lt;a href=&quot;https://f-list.net/c/Kira&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Kira&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;, &lt;/span&gt;&lt;a href=&quot;#USR-Aniko&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Aniko&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;, &lt;/span&gt;&lt;a href=&quot;https://f-list.net/c/-Hexxy&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Hexxy&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;, &lt;/span&gt;&lt;a href=&quot;https://f-list.net/c/Eager&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Eager&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;, &lt;/span&gt;&lt;a href=&quot;https://f-list.net/c/Alicia Sprig&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Alicia Sprig&lt;/span&gt;&lt;/a&gt;,&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt; &lt;/span&gt;&lt;a href=&quot;https://f-list.net/c/Kythyria&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; text-decoration: underline; color:#508ed8;&quot;&gt;Kythyria&lt;/span&gt;&lt;/a&gt; &amp;amp; &lt;a href=&quot;https://www.f-list.net/c/Greyhoof&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Greyhoof&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;&lt;br /&gt;For bug reports, PM Greyhoof or post &lt;/span&gt;&lt;a href=&quot;https://github.com/greyhoof/flist-messenger/issues&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;here&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;.&lt;br /&gt;(Please do not use the helpdesk, contact other staff or the original devs for this if you're using the Hoof Edition - since it is only maintained by Greyhoof.)&lt;br /&gt;&lt;br /&gt;Thank you for using the messenger's beta version. For updates, regularly check the &lt;/span&gt;&lt;a href=&quot;https://github.com/greyhoof/flist-messenger&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;Github page&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-family:'Sans Serif';&quot;&gt;.&lt;br /&gt;&lt;br /&gt;To get debug output, run the application with the &amp;quot;-d&amp;quot; argument.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FAboutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FAboutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
